src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    public static final String DELIMITER = ",";
+    public static final String OPERATION = "buy";
+    private int supplyAmount = 0;
+    private int buyAmount = 0;
```

Comment:
These variables should be local to the method where they are used since they don't represent the state of the class objects. Moving them inside the 'readDataFromTheFile' method would be more appropriate.